march_hardware/include/march_hardware/EtherCAT/EthercatMaster.h

```diff
@@ -27,8 +28,8 @@ class EthercatMaster
   EthercatMaster(const EthercatMaster&) = delete;
   EthercatMaster& operator=(const EthercatMaster&) = delete;
 
-  /* Enable the move constructor */
-  EthercatMaster(EthercatMaster&&) = default;
+  /* Delete move constructor/assignment since atomic bool can not be moved */
```

Comment:
```suggestion
  /* Delete move constructor/assignment since atomic bool cannot be moved */
```
what is a move constructor? :sweat_smile:

Reply:
A move constructor is a constructor which accepts a reference to an instance of its type, then copies all values from that instance and leaves the instance with default values.
https://en.cppreference.com/w/cpp/language/move_constructor